src/core/connection.c

```diff
@@ -6101,6 +6388,20 @@ QuicConnProcessApiOperation(
         ApiCtx->CONN_START.ServerName = NULL;
         break;
 
+    case QUIC_API_TYPE_CONN_SEND_RESUMPTION_TICKET:
+        Status =
+            QuicConnSendResumptionTicket(
+                Connection,
+                ApiCtx->CONN_SEND_RESUMPTION_TICKET.AppDataLength,
+                ApiCtx->CONN_SEND_RESUMPTION_TICKET.ResumptionAppData);
+        ApiCtx->CONN_SEND_RESUMPTION_TICKET.ResumptionAppData = NULL;
+        if (ApiCtx->CONN_SEND_RESUMPTION_TICKET.Flags & QUIC_SEND_RESUMPTION_FLAG_FINAL) {
+            QUIC_DBG_ASSERT(QuicConnIsServer(Connection)); // TODO: handle client-side
```

Comment:
I don't understand this TODO. Why would client ever be sending tickets?

Reply:
I imagined when client-side is implemented it would could reuse the same queued event type.
However, that doesn't make as much sense now as it did then. I'll remove the comment.